stories/components/profile/profile.html.twig

```diff
@@ -1,61 +1,61 @@
-{#
-/**
-  * User Profile
-  *
-  * Available variables:
-  * - variable_name: description.
-**/
-#}
-
-<div class="bix-profile">
-  <div class="bix-profile__main">
-    <div class="bix-profile__main-container">
-      <div class="bix-profile__intro">
-        <div class="bix-profile__image">
-          <img
-            src="{{ image | default("https://placehold.co/325x325") }}"
-            height="270"
-            width="270"
-            alt=""
-          />
-        </div>
-        <div class="bix-profile__description">
-          <h1 class="bix-profile__heading bix-profile__heading--primary">
-            {{ name | default("Full name") }}
-          </h1>
-          <div class="bix-profile__job-description">
-            <div class="bix-profile__role">
-              {{ role | default("Role") }}
-            </div>
-          </div>
-          {% if socialNav %}
-            {% include "@components/social-nav/social-nav.html.twig" with socialNav %}
-          {% endif %}
-        </div>
-      </div>
-    </div>
-  </div>
-
-  {% if specialties or bucketlist %}
-  <div class="bix-profile__aside">
-    <div class="bix-profile__interests">
-      {% if specialties %}
-      <div class="bix-profile__specialties">
-        <h2 class="bix-profile__heading bix-profile__heading--secondary">Specialties</h2>
-        <ul>
-          {% for item in specialties %}
-            <li>{{ item }}</li>
-          {% endfor %}
-        </ul>
-      </div>
-      {% endif %}
-      {% if bucketlist %}
-      <div class="bix-profile__bucket-list">
-        <h2 class="bix-profile__heading bix-profile__heading--secondary">Bucket List</h2>
-        <p>{{ bucketlist }}</p>
-      </div>
-      {% endif %}
-    </div>
-  </div>
-  {% endif %}
-</div>
+{% extends "@components/profile/profile-base.html.twig" %}
+
+{% if image is not empty %}
```

Comment:
This logic won't work for SB. For example, if I delete 'image' from profile.content.json and leave this file as is, there will be no image.
The points of the conditionals here are to not overwrite what comes from profile.content.json. Ie, if image is set there, then it overwrites the defaults set in the 'non-base' file for SB. That should be the case for all these vars here.

Reply:
You can test this by deleting 'image' from the JSON file and going to http://localhost:5050/?path=/story/components-profile--default

Reply:
Fixed!